src/main/fc/settings.c

```diff
@@ -761,6 +761,7 @@ const clivalue_t valueTable[] = {
     { "vtx_band",                   VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, VTX_SETTINGS_MAX_BAND }, PG_VTX_SETTINGS_CONFIG, offsetof(vtxSettingsConfig_t, band) },
     { "vtx_channel",                VAR_UINT8  | MASTER_VALUE, .config.minmax = { VTX_SETTINGS_MIN_CHANNEL, VTX_SETTINGS_MAX_CHANNEL }, PG_VTX_SETTINGS_CONFIG, offsetof(vtxSettingsConfig_t, channel) },
     { "vtx_power",                  VAR_UINT8  | MASTER_VALUE, .config.minmax = { VTX_SETTINGS_MIN_POWER, VTX_SETTINGS_POWER_COUNT }, PG_VTX_SETTINGS_CONFIG, offsetof(vtxSettingsConfig_t, power) },
+    { "vtx_bench_mode",             VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_VTX_SETTINGS_CONFIG, offsetof(vtxSettingsConfig_t, benchMode) },
```

Comment:
Since this will for sure not be used only on the bench, another name might be less confusing.

Reply:
Also it would be cool to have this settable via CMS.

Reply:
I tried to think of something short and simple... vtx_low_power_while_disarmed_mode (and things like it) just seemed too.. wordy?

Reply:
I definitely suck at naming but I would find vtx_low_power_at_disarm more explicit than vtx_bench_mode. It's longer but at least you now directly what it is supposed to do.